src/Native/Bootstrap/main.cpp

```diff
@@ -246,6 +247,20 @@ Object * __get_commandline_args(int argc, char * argv[])
 
     return (Object *)p;
 }
+#else
+Object * __get_commandline_args(int argc, char * argv[])
```

Comment:
I would unify this with the CPPCODEGEN version. Effectively only one line is different.